Library/Homebrew/cask/lib/hbc/audit.rb

```diff
@@ -48,6 +49,22 @@ def summary_header
 
     private
 
+    def check_single_uninstall_zap
+      odebug "Auditing single uninstall_* and zap stanzas"
+
+      uninstalls = cask.artifacts.find_all { |k| k.is_a?(Hbc::Artifact::Uninstall) }
+      add_warning "there should be a single uninstall stanza" unless uninstalls.size <= 1
```

Comment:
Don't use “should” here, maybe something like `"only one uninstall stanza is allowed"`.

Reply:
Done, fix incoming.